plugin-build/src/main/kotlin/io/sentry/android/gradle/autoinstall/jdbc/JdbcInstallStrategy.kt

```diff
@@ -0,0 +1,92 @@
+package io.sentry.android.gradle.autoinstall.jdbc
+
+import io.sentry.android.gradle.SentryPlugin
+import io.sentry.android.gradle.autoinstall.AbstractInstallStrategy
+import io.sentry.android.gradle.autoinstall.AutoInstallState
+import io.sentry.android.gradle.autoinstall.InstallStrategyRegistrar
+import io.sentry.android.gradle.util.SemVer
+import javax.inject.Inject
+import org.gradle.api.artifacts.dsl.ComponentMetadataHandler
+import org.slf4j.Logger
+
+// @CacheableRule
+abstract class JdbcInstallStrategy : AbstractInstallStrategy {
+
+    constructor(logger: Logger) : super() {
+        this.logger = logger
+    }
+
+    @Suppress("unused") // used by Gradle
+    @Inject // inject is needed to avoid Gradle error
+    constructor() : this(SentryPlugin.logger)
+
+    override val sentryModuleId: String get() = SENTRY_JDBC_ID
+
+    override val shouldInstallModule: Boolean get() = AutoInstallState.getInstance().installJdbc
+
+    override val minSupportedThirdPartyVersion: SemVer get() = MIN_SUPPORTED_VERSION
+
+    override val minSupportedSentryVersion: SemVer get() = SemVer(4, 4, 0)
```

Comment:
you probably copy-pasted this, but we should set the min sentry version here, basically when this integration was introduced. E.g. jdbc was introduced in 5.3.0. We need this in case user uses/specifies an older version of the io.sentry:sentry module

We have to change this for all new strategies